### DISCUSSION TITLE
How WinUI3 is implemented?

### DISCUSSION THREAD

**Discussion by ashwith2427** [microsoft/WindowsAppSDK#4985] [Q&A]
Does WinUI3 uses win32 api components beneath it. Like Buttons from windows.h. I saw the source code but iam so confused.

**Comment by castorix** [comment]
It is like UWP  : https://learn.microsoft.com/en-us/windows/uwp/composition/visual-layer
with differences explained at https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/composition
